airflow/secrets/__init__.py

```diff
@@ -50,7 +51,7 @@ def __init__(self, **kwargs):
         pass
 
     @abstractmethod
-    def get_connections(self, conn_id) -> List[Connection]:
+    def get_connections(self, conn_id) -> Optional[List[Connection]]:
```

Comment:
I don't think this should be changed -- return an empty list instead of none?

Reply:
Updated